hack/run-locally.sh

```diff
@@ -0,0 +1,121 @@
+#!/bin/bash 
+
+if [ -z "$CLUSTER_DIR" ]; then
+    echo CLUSTER_DIR is required
+    exit 1
+fi
+
+set -o errexit
+set -o nounset
+
+# Install our overrides so the cluster doesn't run the network operator.
+function override() {
+    local cluster_dir=$1
+    
+    if [ ! -e $cluster_dir/manifests/cvo-overrides.yaml ]; then
+        echo "cannot find cvo-overrides.yaml; please run"
+        echo "openshift-install --dir=$cluster_dir create manifests"
+        exit 1
+    fi
+
+    # Patch the CVO to not create the network operator
+    echo "Applying overrides to $CLUSTER_DIR/manifests/cvo-overrides.yaml"
+
+    kubectl patch --type=json --local=true -f=$CLUSTER_DIR/manifests/cvo-overrides.yaml -p "$(cat hack/overrides-patch.yaml)" -o yaml > $CLUSTER_DIR/manifests/.cvo-overrides.yaml.new
```

Comment:
Even though it doesn't need it, this will fail if you don't have `$KUBECONFIG` pointing to a valid kubeconfig file. Apparently a minimal kubeconfig, which you can point to with `--kubeconfig` is:

    clusters:
    - cluster:
        server: http://localhost

Reply:
Looks like you can even do `KUBECONFIG=/dev/null`

Reply:
I get `error: no configuration has been provided` with that; the validation function appears to want a config containing at least one cluster. (This may depend on your kubectl version?)

Reply:
Weirdly enough, `/dev/null` works for `kubectl patch` and `oc adm release extract`, but not for `oc convert`.  Buhhh.